plugins/terraform/layer0/resource_layer0_service_test.go

```diff
@@ -16,6 +17,8 @@ func TestResourceServiceCreateRead(t *testing.T) {
 
 	mockClient := mock_client.NewMockClient(ctrl)
 
+	defer client.SetTimeMultiplier(0)()
```

Comment:
Is this still needed?

Reply:
Not sure what you mean because I just added it in

Reply:
I'm wondering what does this does when we only deal with mockclient in the test?

Reply:
check out the code we are testing and you'll see where this comes into play

Reply:
Okay I checked it out. The variable name through me off a little.